docs/build-with-powerloom/devnet/getting-started.md

```diff
@@ -145,7 +150,13 @@ Run the command
 ```bash
 bash build.sh
 ```
-Once you start the node, you can check your status from the [devnet dashboard](https://mint-devnet.powerloom.network) or you can check your node logs.
+
```

Comment:
Before this, we need the instructions for installing Python requirements `pip install -r requirements.txt`

Reply:
Added with [863a9fc](https://github.com/PowerLoom/docs/pull/26/commits/863a9fcc0250213e34a5c582c8c8f3f2794b1d74)